tests/cypress/tests-e2e/importToolMaps.cy.js

```diff
@@ -272,16 +272,8 @@ describe('The Import Maps Tool', () => {
 
         //---------------------------------------------------------------------------------
         cy.log('Check that the single layer has been added to the map')
-        // NOTE here below itemId-1 should be present twice, one from the group of layer itemId and
-        // once as single layer
-        cy.checkOlLayer([
-            bgLayer,
-            `${itemId}-1`,
-            `${itemId}-2`,
-            `${itemId}-3`,
-            `${itemId}-1`,
-            singleLayerId,
-        ])
+        // NOTE here below itemId-1 should be present once as a single layer only
```

Comment:
we can remove this comment, it doesn't bring much (was relevant when a layer was twice in the ULR, but now...)